src/mainviews/job-detail.tsx

```diff
@@ -120,201 +96,235 @@ export function JobDetail(props: IJobDetailProps): JSX.Element {
     }
   };
 
-  useEffect(() => {
-    getJob();
-  }, []);
+  const TextFieldStyled = (props: TextFieldProps) => (
+    <TextField {...props} variant="outlined" disabled />
+  );
+
+  const Loading = () => (
+    <Stack direction="row" justifyContent="center">
+      <CircularProgress title={trans.__('Loading')} />
+    </Stack>
+  );
 
-  function TextFieldStyled(props: ITextFieldStyledProps) {
+  const BreadcrumbsStyled = () => (
+    <div role="presentation">
+      <Breadcrumbs aria-label="breadcrumb">
+        <Link
+          underline="hover"
+          color="inherit"
+          onClick={(
+            _:
+              | React.MouseEvent<HTMLAnchorElement, MouseEvent>
+              | React.MouseEvent<HTMLSpanElement, MouseEvent>
+          ): void => props.setView('ListJobs')}
+        >
+          {trans.__('Notebook Jobs')}
+        </Link>
+        <Typography color="text.primary">{props.model.jobName}</Typography>
+      </Breadcrumbs>
+    </div>
+  );
+
+  const ButtonBar = () => (
+    <Stack direction="row" gap={2} justifyContent="flex-end" flexWrap={'wrap'}>
+      {props.model.status === 'IN_PROGRESS' && (
+        <Button variant="outlined" onClick={handleStopJob}>
+          {trans.__('Stop Job')}
+        </Button>
+      )}
+      <Button variant="outlined" onClick={handleRerunJob}>
+        {trans.__('Rerun Job')}
+      </Button>
+      <Button variant="contained" color="error" onClick={handleDeleteJob}>
+        {trans.__('Delete Job')}
+      </Button>
+    </Stack>
+  );
+
+  const coreOptionsFields: TextFieldProps[][] = [
+    [
+      { defaultValue: props.model.jobName, label: trans.__('Job name') },
+      { defaultValue: props.model.jobId, label: trans.__('Job ID') }
+    ],
+    [
+      {
+        defaultValue: props.model.inputFile,
+        label: trans.__('Input file')
+      },
+      {
+        defaultValue: props.model.outputPath,
+        label: trans.__('Output path')
+      }
+    ],
+    [
+      {
+        defaultValue: props.model.environment,
+        label: trans.__('Environment')
+      },
+      { defaultValue: props.model.status ?? '', label: trans.__('Status') }
+    ],
+    [
+      {
+        defaultValue: timestampLocalize(props.model.createTime ?? ''),
+        label: trans.__('Create time')
+      },
+      {
+        defaultValue: timestampLocalize(props.model.updateTime ?? ''),
+        label: trans.__('Update time')
```

Comment:
```suggestion
        label: trans.__('Updated at')
```

Consistency with `Created at` above